acceptance/tests/api-gateway/api_gateway_test.go

```diff
@@ -209,17 +209,25 @@ func TestAPIGateway_Basic(t *testing.T) {
 			checkStatusCondition(t, tcpRoute.Status.Parents[0].Conditions, trueCondition("ConsulAccepted", "Accepted"))
 
 			// check that the Consul entries were created
-			entry, _, err := consulClient.ConfigEntries().Get(api.APIGateway, "gateway", nil)
-			require.NoError(t, err)
-			gateway := entry.(*api.APIGatewayConfigEntry)
+			// On startup, the controller can take upwards of 1m to perform
+			// leader election so we may need to wait a long time for
+			// the reconcile loop to run (hence the 1m timeout here).
```

Comment:
It looks like we have a retry above w/ the same comment - do we expect this second block to fail independently after the first passes for the same leader election, or am I missing some nuance?

Reply:
Removed the comment.